### ISSUE TITLE
A11y: Make the filter more info icons keyboard navigable

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/vacant-lots-proj#328]
## Describe the task
If you hover over the little "i" info icons, you get a helpful tooltip. Currently keyboard users and screenreader users are missing out on that info.

Make the info icon focusable/reachable via keyboard nav. When the user TABs to the icon & lands on it, the tooltip should pop up. 
This should fix things for screenreader users to access the information as well, but we'll have to test it to know.

## Acceptance Criteria

- [ ] On the map page open the filters. 
- [ ] Press TAB button. 
- [ ] You should be able to land on the information icons.
- [ ] The tooltip will pop up. 
- [ ] When you TAB away from the icon, tooltip disappears.

## Links
PR: https://github.com/CodeForPhilly/vacant-lots-proj/pull/350

**Comment by marvieqa** [comment]
Note: 
Adding to the acceptance criteria:
- [ ] When you press ESC, the tooltip disappears.
